README.md

```diff
@@ -18,37 +18,47 @@ DOCKER_COMPOSE=docker-compose -f local-docker-compose.yml
 
 - install [Docker](http://docker.io)
 - bring in ICU source
-
-     cd src
-     git clone --branch master --depth 1 https://github.com/unicode-org/icu.git
-     cd icu
-     git lfs fetch
-     git lfs checkout
-     cd ../..
-
+````
```

Comment:
Also, shouldn't all of these code blocks be indented by 2 spaces so that they are each inside their list bullet, rather than breaking the list?

But no further block-indent needed inside the triple backticks.